content/en/workshop/index.adoc

```diff
@@ -6,69 +6,105 @@ images: []
 
-* Computational linguistics of Turkic languages
-* Evaluation of NLP methods for Turkic languages
-* Building Turkic language NLP systems
-* Metrics and measures for Turkic languages NLP
-* Learning from small or sparse data in Turkic languages
-* Datasets, benchmarks, and software for Turkic languages
+* Computational linguistics: models of all aspects of linguistics in Turkic languages (e.g., semantics, syntax, lexicon, morphology)
+* Systems: Case studies on the construction of NLP systems for Turkic languages
+* Evaluation: Understanding the applicability of current NLP methods in Turkic languages
+* Metrics: New metrics and measures for evaluating NLP systems suitable to Turkic languages
+* Learning from sparse data: Novel methods for learning from small or sparse data in Turkic languages
+* Resources: Datasets, benchmarks, and software libraries for NLP models in Turkic languages
 
-== Submissions Guidelines
+== Hackathon
 
-* Long Papers: Up to 8 pages, excluding references. An additional page is allowed for camera-ready versions. 
-* Extended Abstracts: Up to 2 pages, excluding bibliography. Ideal for ongoing or preliminary research.
-* Submissions should be made through our online submission system.
+The workshop will have a 2-day format and the second day will be devoted to a full-day hybrid hackathon where we will demonstrate the SIGTURK NLP library and interested parties will contribute to the integration of new NLP methods and models into the SIGTURK pipeline. The SIGTURK infrastructure can be found at https://github.com/sigturk. Findings of the hackathon will be combined into a system demonstration paper.
 
-== Awards and Scholarships
+== Submission Guidelines
+
+=== Research papers
+
+We invite all potential participants to submit their novel research contributions in the related fields as long papers following the ACL 2024 long paper format (anonymized with 8 pages excluding the references, and an additional page for the camera-ready versions for the accepted papers). All accepted research papers will be published as part of our workshop proceedings and will be presented either through oral presentations or poster sessions.
+Our research paper track will accept submissions through our own submission system available at https://openreview.net/group?id=aclweb.org/ACL/2024/Workshop/SIGTURK.
 
-* Best Paper Award with a monetary prize.
-* Scholarships for students requiring financial support to participate.
+=== Extended abstracts
 
-== Sponsors
+Besides long paper submissions, we also invite previously published or ongoing and incomplete research contributions to our non-archival extended abstract track. All extended abstracts can use the same EMNLP template with a 2-page limit, excluding the bibliography. Extended abstracts can be submitted to the workshop submission system using the softconf link: https://openreview.net/group?id=aclweb.org/ACL/2024/Workshop/SIGTURK.
```

Comment:
Not SoftConf but OpenReview I guess

Reply:
Fixed in d38000efb03e824265fc31a271f4b661d1dcb738